app/client/apps/goat/components/map/panels/MapStyle.tsx

```diff
@@ -1,9 +1,235 @@
+                  </Typography>
+                  <Checkbox />
+                </CardContent>
+              </Card>
+              {layerAccordionInfo.map((item) => (
+                <Box key={item.id}>
+                  <BasicAccordion title={item.title} variant="secondary" className={classes.accordion}>
+                    <div>Lorem</div>
+                    <div>Lorem</div>
+                    <div>Lorem</div>
+                  </BasicAccordion>
+                  <Divider className={classes.divider} />
+                </Box>
+              ))}
+            </>
+          ) : (
+            <>
+              <Box className={classes.attributeContainer}>
+                <Typography variant="body2">Attribute</Typography>
+                <MultipleSelect options={layerTypes} label="Browser layer attributes" />
+              </Box>
+              <Box>
+                <Text color="secondary" typo="label 2" className={classes.descriptionText}>
+                  Style your map according to the values of a specific attribute or column in the dataset,
+                  using techniques such as color coding or symbol size variation for categorical and numerical
+                  data.
+                </Text>
+              </Box>
+            </>
+          )}
+        </Box>
+      }
+      action={
+        <Box className={classes.buttonsContainer}>
+          <Button className={classes.button} color="secondary" variant="outlined" disabled>
+            Reset
+          </Button>
+          <Button
+            className={classes.button}
+            color="primary"
+            variant="outlined"
+            disabled
+            endIcon={<ArrowDropDownIcon />}>
+            Save As
+          </Button>
+        </Box>
+      }
+    />
+  );
 };
 
+const useStyles = makeStyles({ name: { MapStylePanel } })((theme) => ({
+  contentHeading: {
+    display: "flex",
+    alignItems: "center",
+  },
+  buttonsContainer: {
+    minWidth: "266px",
+    display: "flex",
+    columnGap: "16px",
+  },
+  button: {
+    borderRadius: "24px",
+    textTransform: "none",
+    fontSize: "14px",
+    width: "50%",
+    "&:disabled": {
+      border: "1px solid #ccc",
+      color: "#888",
+    },
+  },
+  contentInfo: {
+    display: "flex",
+    flexDirection: "column",
+    rowGap: "24px",
+  },
+  descriptionText: {
+    fontStyle: "italic",
+    margin: "0  8px",
+  },
+  tab: {
+    width: "50%",
+  },
+  attributeContainer: {
+    display: "flex",
+    flexDirection: "column",
+    rowGap: "8px",
+  },
+  card: {
+    paddingLeft: theme.spacing(2),
+  },
+  radioIcon: {
+    color: "#2BB3814D",
+  },
+  radioIconChecked: {
+    color: "#2BB381",
+  },
+  media: {
+    height: "42px",
+    backgroundColor: "#2bb3810a",
+    border: "none",
+  },
+  content: {
+    display: "flex",
+    columnGap: "6px",
+    padding: "8px 16px",
+  },
+  contentButton: {
+    alignSelf: "flex-start",
+  },
+  accordion: {
```

Comment:
Regarding the backgroundColor/color it will be better to use it from theme.colors.paletter..... so it's easier to change later.